bullet_train-themes-tailwind_css/app/views/themes/tailwind_css/fields/_option.html.erb

```diff
@@ -1,10 +1,19 @@
 <%
 form ||= current_fields_form
+labels = labels_for(form, method) if form
+
+# Initialize options.
```

Comment:
Another spot where we could issue some deprecation warnings and implement some backwards compatibility stuff.